climakitae/selectors.py

```diff
@@ -295,6 +299,57 @@ def _update_location(self):
 
 
 
+# ======================== DATA SELECTIONS ========================
+
+def _get_unique_variables(cat, activity_id, table_id, grid_label): 
+    """Get unique variables for an input catalog subset. 
+    
+    Args: 
+        cat (intake catalog): catalog 
+        activity_id (str): dataset name (i.e. "WRF") 
+        table_id (str): timescale 
+        grid_label (str): resolution
+        
+    Returns: tuple 
+        variable_id_options (list of strs): valid variable id options for input 
+        scenario_options (list of strs): valid scenario (experiment_id) options for input 
+    """
+    # Get catalog subset from user inputs 
+    cat_subset = cat.search(
+        activity_id=activity_id, 
+        table_id=table_id, 
+        grid_label=grid_label
+    )
+    # Get all unique variable id options from catalog selection
+    variable_id_options = cat_subset.unique()["variable_id"]["values"] 
+    
+    # Get all unique scenario options from catalog selection
+    scenario_options = cat_subset.unique()["experiment_id"]["values"] 
+    return variable_id_options, scenario_options
+
+
+def _get_variable_options_df(var_catalog, unique_variable_ids, timescale): 
+    """Get variable information for a subset of unique variable ids. 
+    
+    Args: 
+        var_catalog (df): variable catalog information. read in from csv file 
+        unique_variable_ids (list of strs): list of unique variable ids from catalog. Used to subset var_catalog 
+        timescale (str): hourly, daily, or monthly 
+        
+    Returns: 
+        variable_options_df (pd.DataFrame): var_catalog information subsetted by unique_variable_ids 
+    
+    """
+    if timescale in ["daily","monthly"]: 
+        timescale = "daily/monthly"
```

Comment:
why are these combined as one thing?

Reply:
Because the variables (and therefore the descriptions, units, etc) are the same for both daily and monthly data, while the hourly variables are different